src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ShopUser.xml

```diff
@@ -15,9 +15,26 @@
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xsi:schemaLocation="http://symfony.com/schema/dic/serializer-mapping https://symfony.com/schema/dic/serializer-mapping/serializer-mapping-1.0.xsd"
 >
-    <class name="Sylius\Component\Core\Model\ShopUser">
+    <class name="Sylius\Component\Core\Model\ShopUser" >
+        <attribute name="plainPassword">
+            <group>admin:customer:create</group>
+            <group>admin:customer:update</group>
+        </attribute>
+
+        <attribute name="enabled">
+            <group>admin:customer:create</group>
+            <group>admin:customer:read</group>
+            <group>admin:customer:update</group>
+        </attribute>
+
         <attribute name="verified">
+            <group>admin:customer:read</group>
             <group>shop:customer:read</group>
         </attribute>
+
+        <attribute name="verifiedAt" serialized-name="verified">
```

Comment:
Is it valid to define serialized-name `verified` the same as attribute `verified` above?

Reply:
The returned value is `verified` of type bool, and we want to do the same in the create and update actions to be consistent. The `ShopUser` doesn't have a property like `verified`, only `verifiedAt`, so I use this field for serialization, and in the denormalization process, I convert `true` to a date and `false` to null.